scout/parse/models.py

```diff
@@ -81,7 +82,7 @@ def __eq__(self, other):
 
 class ScoutIndividual(BaseModel):
     alignment_path: Optional[str] = None
-    analysis_type: Literal["wgs", "wes", "mixed", "unknown", "panel", "external"] = None
+    analysis_type: Literal[ANALYSIS_TYPES] = None
```

Comment:
Nice! No issues with testing on that one? I'm wary since we had a problem with pydantic Literal constants a similar construction in CG. 😊

Reply:
I've tested locally on the demo case, but perhaps we can ask @ivadym if they have a real cases that can be loaded in scout stage

Reply:
Yes we can test it with `alertcod`, the scout yaml file has been already generated:`/home/proj/stage/cancer/cases/alertcod/scout_load.yaml`

Reply:
Nice, I'll test it on hasta stage

Reply:
First removed alercod from cust087: 🆗 

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/28093618/173840524-478b0950-92d4-4f07-8fad-977a5d9a4b08.png">

Case loaded OK:

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/28093618/173841278-0fa17f3d-bd9a-46f6-9221-bc3eba8517a7.png">

It works! -> cust087 cases -> https://scout-stage.scilifelab.se/cust087/cases

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/28093618/173841884-938a5d7e-3eb5-4c00-aa88-e6bcb80b6ac0.png">

Specific case -> https://scout-stage.scilifelab.se/cust087/F0046103
<img width="903" alt="image" src="https://user-images.githubusercontent.com/28093618/173841979-9826aaaf-0b2e-42f9-a835-1ef732ce7975.png">

Reply:
Great! Thanks Chiara!